src/pcm.rs

```diff
@@ -923,6 +949,16 @@ impl Status {
     }
 }
 
+/// The type of timestamp returned by `Status::get_audio_htstamp()`.
+/// These values are derived from the [audio_time](https://github.com/alsa-project/alsa-lib/blob/master/test/audio_time.c)
+/// test program in the ALSA library.
+pub enum AudioTstampType {
+    Default = 0,
+    Link = 1,
+    LinkEstimated = 2,
+    Synchronized = 3,
+}
```

Comment:
Probably this enum should be non-exhaustive

Reply:
Agreed. I wasn't sure if there was a MSRV issue so I left it off, but I'll add it.